unit_tests/test_cassandra_stress_thread.py

```diff
@@ -79,6 +79,7 @@ def cleanup_thread():
     assert float(output[0]["latency 99th percentile"]) > 0
 
 
+@pytest.mark.skip("Disabled due to the https://github.com/scylladb/scylla-cluster-tests/issues/9988")
```

Comment:
Could we use skipif instead?

Reply:
I'd require to implement fetching state of the issue or use SkipPerIssue, but I don't know how to use it in context of a unit test decorator.
I think reenabling it upon fixing the issue is good enough.